src/ultra/test/logic.clj

```diff
@@ -0,0 +1,44 @@
+(ns ultra.test.logic
+  (:require [clojure.test :as test]
+            [clojure.string :as s]
+            [ultra.printer :refer [cprint]]
+            [ultra.test.diff :refer [pretty]]))
+
+(def logic-ops
```

Comment:
Please add `:added` metadata to all new functions. Helps to track breaking changes.